src/BlockEntities/BrewingstandEntity.h

```diff
@@ -119,6 +126,9 @@ class cBrewingstandEntity :
 	/** Amount of ticks that the current item has been brewed */
 	short m_TimeBrewed;
 
+	/** The fuel for the brewing stand */
+	short m_Fuel;
```

Comment:
Okay, but we still need a more descriptive variable name, so that it's clear that it's not the actual item, but some measure of the fuel left. `m_FuelLeft` would be okayish with me, if we can't think of anything better.